src/entries/popup/components/SwitchMenu/SwitchNetworkMenu.tsx

```diff
@@ -106,9 +106,14 @@ export const SwitchNetworkMenuSelector = ({
     handler: handleTokenShortcuts,
   });
 
+  const supportedChainsWithHardhat = useMemo(
+    () => getSupportedChainsWithHardhat(),
+    [],
+  );
+
   return (
     <Box id="switch-network-menu-selector">
-      {getSupportedChainsWithHardhat().map((chain, i) => {
+      {supportedChainsWithHardhat.map((chain, i) => {
```

Comment:
does wrapping it in useMemo change anything?

Reply:
hmm probably not

Reply:
okok just updated this 👌